src/CP/variables/BoolVarView.jl

```diff
@@ -43,6 +43,8 @@ Return the size of `dom`.
 """
 Base.length(dom::BoolDomainView) = length(dom.orig)
 
+id(dom::BoolVarView) = id(dom.x)
+

```

Comment:
fix a missing required function

Reply:
This isn't coherent, as x already has a field `id`
To get the parent id use `rootVariable(x).id`